test/helpers/chainTest.ts

```diff
@@ -34,17 +35,17 @@ const submitRunProcessExtrinsicAndSeal = async (
 }
 
 export const withAppAndIndexer = (context: { app: Express; indexer: Indexer }) => {
```

Comment:
basically none of this worked. We somehow lost the trim of the blockhash so the insert just failed. Also this wasn't running for each test so one failure was causing all the tests to fail